### PR TITLE
bugfix/sd-112: updated footer to match design.

### PR DESCRIPTION
**Pull request by MasseyIsaako**

### CODE REVIEW

File: dist/css/main.css [hunk 1]
```diff
@@ -15697,24 +15697,37 @@ body.theme-footer-white .footer-site hr {
...
 
+@media (min-width: 992px) {
+  .footer-govt-logo a {
+    margin: 0;
+  }
+}
+
 .footer-nav-links a {
-  color: white;
+  color: white !important;
```
Comment (micmania1) [inline dist/css/main.css]: Why do we need to use `!important` here?

Comment (MasseyIsaako) [reply dist/css/main.css]: screen shot 2018-12-11 at 3 36 14 pm

The `color: white;` property is being overridden by the `theme-picker` from the user. The `!important` is a quick fix, but I'm open to better solutions :)

File: src/scss/components/footer.scss [hunk 2]
```diff
@@ -27,19 +27,27 @@
...
+
+  @include media-breakpoint-up(lg) {
+    margin: 0;
+  }
 }
 
 // Footer navigation and Copyright
 .footer-nav-links a {
-  color: $navbar-dark-color;
+  color: $navbar-dark-color !important;
```
Comment (micmania1) [inline src/scss/components/footer.scss]: Same here @MasseyIsaako

Comment (MasseyIsaako) [reply src/scss/components/footer.scss]: Done!

File: dist/css/main.css (hunk 1 above)
Comment (MasseyIsaako) [reply dist/css/main.css]: Done!

**Comment by micmania1** [comment]
@MasseyIsaako this is how the footer looks for me:

image

The border is missing and the links are the wrong colour (should be white).

**Comment by MasseyIsaako** [comment]
@micmania1 That's really weird, I just ran a flush in my code change branch and it is appearing fine . . . Did you pull down the latest changes? It should be looking like this:

screen shot 2018-12-12 at 11 48 30 am

**Comment by micmania1** [comment]
@MasseyIsaako I think the problem was with my DB. I changed the footer colour from default to something else and then back to default and it now works :man_facepalming:

**Pull request merged**